.github/workflows/ci.yml

```diff
@@ -40,6 +40,11 @@ jobs:
     with:
       additional-artifact-content-paths: |
         archive/output000/MOM_parameter_doc.*
+        archive/output000/*.err
+        archive/output000/*.log
+        archive/output000/job.yaml
+        archive/output000/env.yaml
+        archive/output000/log
```

Comment:
Will this get the archive/output000/log folder ?

Reply:
Yep!